main/ejml-core/src/org/ejml/data/DMatrix.java

```diff
@@ -73,5 +72,7 @@
      *
      * @return Number of elements in this matrix.
      */
-    int getNumElements();
+    default int getNumElements() {
+        return getNumRows() * getNumCols();
```

Comment:
Good idea. I had double check since I was thinking this feature came after java 8, but nope it's a Java 8 feature.